resources/theme.js

```diff
@@ -1,33 +1,66 @@
 var theme = localStorage.getItem("siteTheme");
 var themeButton = document.getElementById("changeTheme");
-themeButton.innerHTML = "Change Theme (" + theme + ")";
 
 if (theme == null){
+	theme = "system";
+}
+
+themeButton.innerHTML = "Change Theme (" + theme + ")";
```

Comment:
over here would ${} work?

Reply:
or really ${theme}

Reply:
Yes, it would be better. I've changed it!